iOS/docs/kickoff/README.md

```diff
@@ -107,6 +107,25 @@ Check the value for `REQUIRED_XCODE_VERSION` matches the installed version. You
 Replace the values for `REQUIRED_SWIFTLINT_VERSION` and `REQUIRED_CARTHAGE_VERSION` with the last release version for each of them.
 You can check them in [SwiftLint releases](https://github.com/realm/SwiftLint/releases) and [Carthage releases](https://github.com/Carthage/Carthage/releases).
 
+#### Include Fastlane
+
+Adding `Fastlane` to the project requires 2 steps, adding the gem, and downloading the scripts.
+
+First, add this to `Gemfile`:
+
+```
+# Fastlane
+gem 'fastlane', '~>2.62.0'
+```
+
+And run `bundle install` to get the `Fastlane` gem.
+
+Then, download the fastlane content from the [Fastlane repository](https://github.com/Wolox/fastlane-mobile) by doing (be careful to be in project root when running this command):
```

Comment:
(IMHO) I'd write it like this for better readability:

"Move to the project's root folder and download the fastlane content from the [Fastlane repository](https://github.com/Wolox/fastlane-mobile) by doing:"